### PR TITLE
testing smoke tests from a fork

### PR DESCRIPTION
**Pull request by jake-inverted**

### CODE REVIEW

File: common/.rubocop.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 inherit_from: ../.rubocop.yml
+
```
Comment (landongrindheim) [inline common/.rubocop.yml:2]:
````suggestion
````

**Review by landongrindheim — CHANGES_REQUESTED** [review]
I'm hoping @jakecoffman can take a look at these before we approve 😇

**Comment by jakecoffman** [comment]
it me

**Comment by landongrindheim** [comment]
> it me

**Pull request closed without merging**